ompi/mca/topo/treematch/treematch/tm_topology.c

```diff
@@ -141,7 +141,12 @@ double ** topology_to_arch(hwloc_topology_t topology)
   double **arch = NULL;
 
   nb_proc = hwloc_get_nbobjs_by_type(topology, HWLOC_OBJ_PU);
+  if( nb_proc <= 0 )  /* if multiple levels with PUs */
+      return NULL;
   arch = (double**)MALLOC(sizeof(double*)*nb_proc);
+  if( NULL == arch )
```

Comment:
Minor: both of these blocks don't obey our code standards of always including blocks -- even 1-line blocks -- in `{}`.  😜

Reply:
Based on the code accepted lately I was under the impression that we are not following the code standard anymore...

Reply:
That and 4-space tabs are about the only thing we ever enforced from the code standards (minimal that they were)...  Maybe putting `{` in the same line as `if`/`while`/etc., too.